test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -580,4 +580,41 @@ describe('Test gl plot side effects', function() {
             });
         });
     });
+
+    it('should be able to replot from a blank graph', function(done) {
+        var gd = createGraphDiv();
+
+        function assert(cnt) {
```

Comment:
Maybe change the name of this. I know it's not really node code, but at a glance without seeing the function, it could be mistaken for the node [assert](https://nodejs.org/api/assert.html).